### DISCUSSION TITLE
Sens de l'attaque

### DISCUSSION THREAD

**Discussion by philippejuhel** [robot-soccer-kit/robot-soccer-kit#101] [Q&A] [answered]
Bonjour,

Une question simple: comment sait-on dans quel sens un robot doit progresser pour attaquer? Il doit bien y avoir une variable indiquant si les bleus doivent marquer dans le but de gauche ou dans le but de droite. Laquelle?

Cordialement,

Philippe

**Comment by Gregwar** [comment, marked as answer]
Bonjour,

Dans la structure [referee](https://robot-soccer-kit.github.io/referee), vous pourrez accéder procéduralement aux informations de l'arbitrage (noms des équipes, score, robots pénalisés etc.), la valeur de `x_positive` vous donnera l'information que vous cherchez
